project/BinaryIncompatibilities.scala

```diff
@@ -8,6 +8,10 @@ object BinaryIncompatibilities {
   )
 
   val Linker = Seq(
+      // New method in sealed trait, not an issue.
```

Comment:
I suggest always keeping the "breaking" exclusions on top (even the "OK" ones), and have the "not an issue" exclusions after them.